code/crates/test/proto/sync.proto

```diff
@@ -17,11 +17,12 @@ message Status {
 
 message ValueRequest {
     uint64 height = 1;
+    optional uint64 end_height = 2;
 }
 
 message ValueResponse {
-    uint64 height = 1;
-    SyncedValue value = 2;
+    uint64 start_height = 1;
```

Comment:
We can safely rename the fields because the binary format is compatible as long as we keep the same tag number.